pkg/probe/http/http.go

```diff
@@ -113,8 +115,12 @@ func DoHTTPProbe(req *http.Request, client GetHTTPInterface) (probe.Result, stri
 			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
 			return probe.Warning, fmt.Sprintf("Probe terminated redirects, Response body: %v", body), nil
 		}
-		klog.V(4).Infof("Probe succeeded for %s, Response: %v", url.String(), *res)
-		return probe.Success, body, nil
+		if limitReached {
+			return probe.Warning, fmt.Sprintf("Non fatal body truncation for %s, Response: %v", url.String(), body), nil
+		} else {
+			klog.V(4).Infof("Probe succeeded for %s, Response: %v", url.String(), *res)
+			return probe.Success, body, nil
+		}
```

Comment:
if ends with return statement, so remove unnecessary else.
And do you think return `response` is better for troubleshooting than return `body`?
```suggestion
		if limitReached {
			return probe.Warning, fmt.Sprintf("Non fatal body truncation for %s, Response: %v", url.String(), body), nil
		} 
		klog.V(4).Infof("Probe succeeded for %s, Response: %v", url.String(), *res)
		return probe.Success, body, nil	
```